README.md

```diff
@@ -1 +1,47 @@
-# Test-To-Do
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: TodoList Testing
+
+> Description the project: With the design of this project the user can add todo list and can set it as completed or not. it's easy to use and simple ;)
+
+
+## Built With
+
+- Major languages
+> Html amd JavaScript
+
+
+## Authors
+
+👤 **Sajjad Rajaee**
+
+- GitHub: [@sajjadrajaee](https://github.com/sajjadrajaee)
+- Twitter: [@sajjadrajaee](https://twitter.com/sajjadrajaee)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/sajjadrajaee)
+
+👤 **Aleksandra Ujvari**
+
+- GitHub: [@Github](https://github.com/digia3d)
+- Twitter: [@twitter](https://twitter.com/ujvari65)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/aleksandra-ujvari-85235a210/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- Please, kindly add the MIT.md licensed content as the link is broken to make your readme more professional.